helm/prometheus-operator-app/README.md

```diff
@@ -166,7 +166,7 @@ The same chart can be used to run multiple Prometheus instances in the same clus
 
 ### Running on private GKE clusters
 
-When Google configure the control plane for private clusters, they automatically configure VPC peering between your Kubernetes cluster’s network and a separate Google managed project. In order to restrict what Google are able to access within your cluster, the firewall rules configured restrict access to your Kubernetes pods. This means that in order to use the webhook component with a GKE private cluster, you must configure an additional firewall rule to allow the GKE control plane access to your webhook pod.
+When Google configures the control plane for private clusters, they automatically configure VPC peering between your Kubernetes cluster’s network and a separate Google managed project. In order to restrict what Google are able to access within your cluster, the firewall rules configured restrict access to your Kubernetes pods. This means that in order to use the webhook component with a GKE private cluster, you must configure an additional firewall rule to allow the GKE control plane access to your webhook pod.
```

Comment:
Should we change `control plane` to `management cluster` or `management api`? I don't know if this is ours or GKE...